### PR TITLE
Workaround guest beatmaps not showing on user profile

### PR DESCRIPTION
**Pull request by notbakaneko**
For the unmigrated entries or anything not updating the new table yet.

~~I was going to update another fallback but it seems to have introduced a display bug for some reason so I'll make that separate~~ 👀 
nvm it's just wrong method

### CODE REVIEW

**Comment by nanaya** [comment]
I'm running the migration so the fallback query is probably not needed.

**Comment by notbakaneko** [comment]
I was thinking new submissions but that can't have guests set during submissions....or can they? 🤔

**Comment by nanaya** [comment]
I don't think that's possible, no.

File: app/Models/Beatmap.php [hunk 1]
```diff
@@ -298,9 +298,10 @@ public function getOwners(): Collection
             fn ($beatmapOwner) => $beatmapOwner->user ?? new DeletedUser(['user_id' => $beatmapOwner->user_id])
         );
 
+        // Fallback if unpopulated
         // TODO: remove when everything writes to beatmap_owners.
```
Comment (nanaya) [inline app/Models/Beatmap.php:302]: not sure the pr title is relevant anymore. also this todo should be adjusted/removed depending on how empty owners should be handled

**Comment by notbakaneko** [comment]
I will think about it later 💤

**Pull request closed without merging**